samples/cpp/benchmark_app/main.cpp

```diff
@@ -909,7 +905,17 @@ int main(int argc, char* argv[]) {
         // ----------------------------------------
         next_step();
 
+        auto create_requests_start_time = Time::now();
         InferRequestsQueue inferRequestsQueue(compiledModel, nireq, app_inputs_info.size(), FLAGS_pcseq);
+        auto create_requests_duration_ms = get_duration_ms_till_now(create_requests_start_time);
+        slog::info << "Creating " << nireq << " infer requests took " << double_to_string(create_requests_duration_ms)
+                   << " ms" << slog::endl;
```

Comment:
Is it actually needed?

I suggest removing it from python instead

Reply:
You're right, it is unnecessary. Removed from both versions.